### PR TITLE
Namespace fixes

### PR DESCRIPTION
**Pull request by florinciubotariu**
Fixed namespaces in multiple files.
Still trying to find out how to correctly use VS Git extension. Sorry if there are problems. Any tips would be appreciated.
After creating a branch in VS, make changes, Commit that branch and pull-request is accepted, how can I synchronize my forked develop branch (that hasn't my changes because the commit was done to another branch) with the upstream develop branch using the VS?

### CODE REVIEW

**Comment by JanOuborny** [comment]
I just hard reset the pulled commits and then rebase onto the upstream `develop `branch and force push it to my forked repository. VS git support is pretty thrashy at the moment, therefore I would use the command line for that.

**Comment by florinciubotariu** [comment]
I did something like
`git reset --hard upstream-name-here/develop`
Then
`git push origin develop --force`
It seems we are doing the same thing. Thank you @JacobSnyder .

**Comment by JanOuborny** [comment]
In your case you forgot to merge or rebase, but basically it's the same.

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\RiotSharp"
+  ],
+  "SelectedNode": "\\RiotSharp\\RiotApi.cs",
+  "PreviewInSolutionExplorer": false
+}
```
Comment (BenFradet) [inline .vs/VSWorkspaceState.json]: Could you remove the `.vs` files from the commit?

**Comment by florinciubotariu** [comment]
@BenFradet  Is https://github.com/florinciubotariu/RiotSharp/commit/3f9f9b96490a56ab2000c78e3ee5dd591d688316 good?

**Comment by BenFradet** [comment]
@florinciubotariu I'll have a look soon 👍

File: RiotSharp/IRiotApi.cs [hunk 2]
```diff
@@ -10,7 +10,8 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using System;
-using RiotSharp.MatchListEndpoint;
+using RiotSharp.MatchListEndPoint;
```
Comment (BenFradet) [inline RiotSharp/IRiotApi.cs]: Shouldn't this be `Endpoint`?

File: RiotSharp/IStaticRiotApi.cs [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
 using System.Threading.Tasks;
+using RiotSharp.Misc;
+using RiotSharp.StaticDataEndpoint.Champion;
+using RiotSharp.StaticDataEndpoint.Item;
+using RiotSharp.StaticDataEndpoint.LanguageStrings;
+using RiotSharp.StaticDataEndpoint.Map;
+using RiotSharp.StaticDataEndpoint.Mastery;
+using RiotSharp.StaticDataEndpoint.Realm;
+using RiotSharp.StaticDataEndpoint.Rune;
+using RiotSharp.StaticDataEndpoint.SummonerSpell;
```
Comment (BenFradet) [inline RiotSharp/IStaticRiotApi.cs]: why not just `using RiotSharp.StaticDataEndpoint;`?

File: RiotSharp/MatchListEndPoint/MatchList.cs [hunk 4]
```diff
@@ -1,7 +1,7 @@
-﻿using Newtonsoft.Json;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
 
-namespace RiotSharp.MatchListEndpoint
+namespace RiotSharp.MatchListEndPoint
```
Comment (BenFradet) [inline RiotSharp/MatchListEndPoint/MatchList.cs]: I think `MatchListEndpoint` is fine

File: RiotSharp/MatchListEndPoint/MatchReference.cs [hunk 5]
```diff
@@ -1,8 +1,10 @@
-﻿using Newtonsoft.Json;
+﻿using System;
+using Newtonsoft.Json;
 using RiotSharp.MatchEndpoint.Enums;
-using System;
+using RiotSharp.Misc;
+using RiotSharp.Misc.Converters;
 
-namespace RiotSharp.MatchListEndpoint
+namespace RiotSharp.MatchListEndPoint
```
Comment (BenFradet) [inline RiotSharp/MatchListEndPoint/MatchReference.cs]: same

File: RiotSharp/MatchListEndPoint/Queue.cs [hunk 6]
```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace RiotSharp
+﻿namespace RiotSharp.MatchListEndPoint
```
Comment (BenFradet) [inline RiotSharp/MatchListEndPoint/Queue.cs]: same

File: RiotSharp/RiotApi.cs [hunk 7]
```diff
@@ -6,13 +6,15 @@
...
 using RiotSharp.LeagueEndpoint;
 using RiotSharp.MatchEndpoint;
-using RiotSharp.MatchListEndpoint;
 using RiotSharp.StatsEndpoint;
 using RiotSharp.SummonerEndpoint;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using RiotSharp.MatchListEndPoint;
```
Comment (BenFradet) [inline RiotSharp/RiotApi.cs]: same

File: RiotSharp/StaticDataEndpoint/BasicDataStatic.cs [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ﻿using Newtonsoft.Json;
 using System.Collections.Generic;
+using RiotSharp.StaticDataEndpoint.Item;
```
Comment (BenFradet) [inline RiotSharp/StaticDataEndpoint/BasicDataStatic.cs]: is this needed?

File: RiotSharp/StaticRiotApi.cs [hunk 9]
```diff
@@ -4,6 +4,23 @@
...
+using RiotSharp.StaticDataEndpoint.Map;
+using RiotSharp.StaticDataEndpoint.Map.Cache;
+using RiotSharp.StaticDataEndpoint.Mastery;
+using RiotSharp.StaticDataEndpoint.Mastery.Cache;
+using RiotSharp.StaticDataEndpoint.Realm;
+using RiotSharp.StaticDataEndpoint.Realm.Cache;
+using RiotSharp.StaticDataEndpoint.Rune;
+using RiotSharp.StaticDataEndpoint.Rune.Cache;
+using RiotSharp.StaticDataEndpoint.SummonerSpell;
+using RiotSharp.StaticDataEndpoint.SummonerSpell.Cache;
```
Comment (BenFradet) [inline RiotSharp/StaticRiotApi.cs]: Let's reduce this

File: RiotSharp/IStaticRiotApi.cs (hunk 3 above)
Comment (florinciubotariu) [inline RiotSharp/IStaticRiotApi.cs]: Mastery, Realm, Rune, SummonerSpell are subfolders in StaticDataEndpoint.

**Comment by florinciubotariu** [comment]
@BenFradet 
https://github.com/florinciubotariu/RiotSharp/commit/0165dcbfb7e4a073f419a70cc22c5defb02a9f5e Fixes for "MatchListEndPoint" renaming.
About the simplifyng multiple namespace into one, we can't do that if we want to keep namespaces according to the project structure.

**Comment by BenFradet** [comment]
Yup my bad didn't see that there were referring to different namespaces cache vs non-cache.

**Comment by florinciubotariu** [comment]
If I omitted any fixes in the last commit, please write them down.

**Pull request merged**